Content.Server/Medical/SuitSensors/SuitSensorSystem.cs

```diff
@@ -228,7 +228,7 @@ private string GetModeName(SuitSensorMode mode)
                 return null;
 
             // check if sensor is enabled and worn by user
-            if (sensor.Mode == SuitSensorMode.SensorOff || sensor.User == null)
+            if (sensor.Mode == SuitSensorMode.SensorOff || sensor.User == null || transform.GridUid == null)
```

Comment:
Don't sensors still work in space in ss13?

Reply:
IIRC this threw some sort of error if I didn't check for a null grid.